### ISSUE TITLE
Impaired gastric accommodation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11675]
Impaired gastric accommodation

## Proposed definition

Reduced capacity of the stomach, particularly the fundus, to relax and expand in response to a meal. PMID:34337824 PMID:16042912

## Parent term

- `HP:0030895 Abnormal gastrointestinal motility`

## Synonyms

- Reduced gastric accommodation
- Impaired fundic accommodation

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001517
- label: Impaired gastric accommodation
- parents: [Abnormal gastrointestinal motility (HP:0030895)](https://hpo.jax.org/app/browse/term/HP:0030895)
- pmid: PMID:34337824|PMID:16042912
- def: Reduced capacity of the stomach, particularly the fundus, to relax and expand in response to a meal.
- comment: Gastric accommodation to a meal consists of relaxation of the proximal stomach without a concomitant increase in intragastric pressure. The gastric accommodation response consists of two primary and complementary motor functions during meal consumption: receptive relaxation and adaptive relaxation.
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11675
- synonyms:
